mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -678,6 +682,7 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 			ARG 1 color
 		METHOD m_yetkcise breakInstantly ()Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
 			COMMENT Specifies that a block is broken instantly.
+		METHOD m_yeubaeen noBlockBreakParticles ()Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
```

Comment:
```suggestion
		METHOD m_yeubaeen disableParticlesOnBreak ()Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
```

Reply:
ah, i was worried about breaking some sort of informal convention by using "disable" instead of "no"